client/src/components/gameShared/NotificationCount.vue

```diff
@@ -1,7 +1,8 @@
 <template>
   <span
     :style="cssProps"
-    class="rounded-pill badge bg-burgundy d-inline-flex align-items-center justify-content-center p-0">
+    class="rounded-pill badge d-inline-flex align-items-center justify-content-center p-0"
+    :class="{ 'bg-blue': !nextMajority, 'bg-orange': nextMajority }">
```

Comment:
don't love the blue on blue, I think burgundy was fine but if you want to experiment with other colours that's ok. Just make sure there is some colour contrast.

Reply:
Tested burgundy and it looks good as the default before transitioning to orange if action occurs on the next vote.